Source/WebGPU/WebGPU/Texture.mm

```diff
@@ -2380,21 +2381,25 @@ static WGPUExtent3D computeRenderExtent(const WGPUExtent3D& baseSize, uint32_t m
 
     auto logicalExtent = logicalMiplevelSpecificTextureExtent(mipLevel);
 
+    auto roundUpToMultipleOfNonPowerOfTwo = ^(size_t a, size_t b) {
+        return static_cast<uint32_t>(bmalloc::roundUpToMultipleOfNonPowerOfTwo(a, b));
+    };
+
     switch (m_dimension) {
     case WGPUTextureDimension_1D:
         return {
-            static_cast<uint32_t>(WTF::roundUpToMultipleOf(texelBlockWidth(m_format), logicalExtent.width)),
+            roundUpToMultipleOfNonPowerOfTwo(texelBlockWidth(m_format), logicalExtent.width),
```

Comment:
The commit message says:

> we can't use WTF::roundUpToMultipleOfNonPowerOfTwo

Yet it looks like we're explicitly using it here?

What happens with texelBlockWidth(m_format) _is_ a power of 2? From the name alone, it sounds like `roundUpToMultipleOfNonPowerOfTwo` would break somehow? 🤔

Reply:
oh commit message should say `WTF::roundUpToMultipleOf`, I'll fix that.

The implementation of `roundUpToMultipleOfNonPowerOfTwo(a, b)` is:

(b + a - 1) / a * a

assuming `a` and `b` are integers, it works despite `a` being a power of two, I will make that more clear